content/blog/gifts-q3-2020.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Gifts Q3 2020"
+date = 2020-09-22
+description = "Twice a year we distribute address space to those that have made valuable contributions to Urbit. Now called our *Gifts* program, the gifting of address space has been part of Urbit long before we had a grants program."
+[extra]
+author = "Josh Lehman"
+ship = "~wolref-podlex"
+image ="https://media.urbit.org/site/posts/essays/gifts.png"
++++
+
+<br>
+
+<img class="ba" src="https://media.urbit.org/site/posts/essays/gifts.png">
+
+<br>
+
+Twice a year we distribute address space to those that have made valuable contributions to Urbit. Now called our *Gifts* program, the gifting of address space has been part of Urbit long before we had a grants program.
+
+I'm happy to announce that urbit.org has a record number of gifts to distribute to community contributors for the period from February to August of 2020. Because the gifting of address space is an old tradition, and the last several months have seen an unprecedented amount of user growth, let’s talk about why we do this.
+
+## Why gifts?
+One way to think about Urbit is as *infrastructure for powering digital communities.* We’ve talked about this before in [some detail](https://urbit.org/blog/urbit-is-for-communities/), so while I won’t rehash the “why” of that here, I’ll reiterate that:
+
+> What we really want is a digital world that’s generated by the communities that inhabit it
```

Comment:
Just personal taste here — this could just be placed as the end of the sentence and it would look more natural. 

```
One way to think about Urbit is as *infrastructure for powering digital communities.* We’ve talked about this before in [some detail](https://urbit.org/blog/urbit-is-for-communities/), so while I won’t rehash the “why” of that here, I’ll reiterate that what we really want is a digital world that's generated by the communities that inhabit it.
```